src/ui/screens/Parachains/AuctionsScreen.tsx

```diff
@@ -51,81 +53,85 @@ export function AuctionsScreen() {
+                  {endingPeriod && (
+                    <ProgressChartWidget
+                      title={`Ending period`}
+                      detail={`${remainingPercent}%\n${endingPeriod.remaining.slice(0, 2).join('\n')}`}
+                      data={[remainingPercent / 100]}
+                    />
+                  )}
                 </View>
-              }
-              right={() => (
-                <View style={globalStyles.justifyCenter}>
-                  <Caption>{`Leases`}</Caption>
-                  <Caption>{`${winningBid?.firstSlot} - ${winningBid?.lastSlot}`}</Caption>
+                <View style={globalStyles.flex}>
+                  <ProgressChartWidget
+                    title={`Total Raised`}
+                    detail={`${raisedPercent}%\n${raisedFormatted.split(' ').join('\n')}`}
+                    data={[raisedPercent / 100]}
+                  />
                 </View>
-              )}
-            />
-          ) : (
-            <>
-              <Padder scale={1} />
-              <Caption style={globalStyles.textCenter}>{`There is no winning bid currently`}</Caption>
-            </>
-          )
-        ) : (
-          <>
-            <Padder scale={1} />
-            <Caption style={globalStyles.textCenter}>{`The auction is not active`}</Caption>
-          </>
-        )}
-      </Card>
+              </View>
+            </Card.Content>
+          </Card>
+
+          <Card style={styles.container}>
+            {winningBid ? (
+              <>
+                <Subheading style={globalStyles.textCenter}>{`Winning Bid`}</Subheading>
+                <List.Item
+                  title={winningBid?.projectName}
+                  description={
+                    <View>
+                      <Padder scale={0.5} />
+                      <Caption>{`ProjectID: ${winningBid?.projectId}`}</Caption>
+                      <Caption>{`Bid: ${winningBid?.amount} ${winningBid?.isCrowdloan ? '(crowdloan)' : ''}`}</Caption>
+                      <Caption>
+                        {`Block number: `}
+                        <Caption onPress={copyToClipboard}>{winningBid?.blockNumber}</Caption>
+                      </Caption>
+                    </View>
+                  }
+                  right={() => (
+                    <View style={globalStyles.justifyCenter}>
+                      <Caption>{`Leases`}</Caption>
+                      <Caption>{`${winningBid?.firstSlot} - ${winningBid?.lastSlot}`}</Caption>
+                    </View>
+                  )}
+                />
+              </>
+            ) : (
+              <Card.Content>
+                <EmptyStateTeaser subheading="There is no winning bid" />
+              </Card.Content>
+            )}
+          </Card>
+        </>
+      ) : (
+        <Card style={styles.container}>
+          <Card.Content>
+            <EmptyStateTeaser subheading="Auction is not active" />
```

Comment:
For this case would be better if we use the empty state for the entire screen (the same we used for my accounts for example) but with this text